rawls-client/build.gradle

```diff
@@ -0,0 +1,16 @@
+import org.springframework.boot.gradle.plugin.SpringBootPlugin
+
+plugins {
+    id 'bio.terra.pipelines.java-library-conventions'
+    id 'io.spring.dependency-management'
+    id 'org.hidetake.swagger.generator'
+    id 'de.undercouch.download'
```

Comment:
i was curious what this does, looks like it's a progress meter for downloads - is this actually used somewhere? when i run locally i don't see anything that looks like it's using this, just: 

```
> Task :rawls-client:downloadRawlsYaml
Download https://raw.githubusercontent.com/broadinstitute/rawls/874b2e1bc6d95f90e1cf0db7569cfbaae9da34bf/core/src/main/resources/swagger/api-docs.yaml
```

Reply:
so the download command is using that plugin (if you remove the plugin the download wont be able to work).  I think the reason you arent seeing the progress is because it seems to dissapear after its done - https://github.com/michel-kraemer/gradle-download-task/blob/master/screencast/screencast.svg